test/singlenode_test.go

```diff
@@ -358,7 +358,7 @@ func TestSingleNodeUpdateParticipant(t *testing.T) {
 		require.True(t, errors.As(err, &twErr))
 		// Note: for Cloud this would return 404, currently we are not able to differentiate between
 		// non-existent participant vs server being unavailable in OSS
-		require.Equal(t, twirp.Unavailable, twErr.Code())
+		require.Equal(t, twirp.NotFound, twErr.Code())
```

Comment:
Requesting @paulwe for review. If this change looks good, the comment above should be removed.